### PR TITLE
feat: add responsivity to conversation view acc-63

### PR DESCRIPTION
**Pull request by V-Gira**
<table>
<td>
  <a href="https://wearezeta.atlassian.net/browse/ACC-63" title="ACC-63" target="_blank">SubtaskACC-63</a>  Conversations View
  </td></table>
  <br />
 

### Make conversation view responsive

- add breakpoints to titlebar, inputbar and conversation view
- display ui-element according to figma reference for scaled down view

Peek 2022-09-27 17-36

### CODE REVIEW

File: src/script/components/InputBar/index.tsx [hunk 1]
```diff
@@ -861,20 +871,70 @@ const InputBar = ({
...
+                          input={inputValue}
+                          isEditing={isEditing}
+                          isScaledDown={isScaledDown}
+                          disablePing={pingDisabled}
+                          disableFilesharing={!isFileSharingSendingEnabled}
+                          onSelectFiles={uploadFiles}
+                          onSelectImages={uploadImages}
+                          onCancelEditing={cancelMessageEditing}
+                          onGifClick={onGifClick}
+                          onClickPing={onPingClick}
```
Comment (atomrc) [inline src/script/components/InputBar/index.tsx]: Quite a bit of duplicated code between the 2 conditions. 
let's try to extract the component instantiation to variables and then place them according to the layout. 

something like:

```jsx
const sendButton = <button ...><Icon.Send /></button>;
const controlButtons = <ControlButtons ... />

return ...
isScaledDown ? <ul>{sendButton}</ul><ul>{controlButtons}</ul> : <ul>{controlButtons}</ul><ul>{sendButton></ul>
```

Comment (V-Gira) [reply src/script/components/InputBar/index.tsx]: got something like this:

```
  const controlButtonsProps = {
    conversation: conversationEntity,
    disableFilesharing: isFileSharingSendingEnabled,
    disablePing: pingDisabled,
    input: inputValue,
    isEditing: isEditing,
    onCancelEditing: cancelMessageEditing,
    onClickPing: onPingClick,
    onGifClick: onGifClick,
    onSelectFiles: uploadFiles,
    onSelectImages: uploadImages,
  };
```

```
                    <ul className="controls-right buttons-group">
                      <ControlButtons {...controlButtonsProps} showGiphyButton={showGiphyButton} />
                      {sendButton}
                    </ul>
```

File: src/script/components/TitleBar/index.test.tsx [hunk 2]
```diff
@@ -137,6 +145,8 @@ describe('TitleBar', () => {
     const panelViewModel = createPanelViewModel();
     const conversation = createConversationEntity();
 
+    mockedUiKit.useMatchMedia.mockReturnValue(false);
```
Comment (atomrc) [inline src/script/components/TitleBar/index.test.tsx:148]: challenge for you: write a test that checks that the component renders correctly when the view is scaledDown 😉

**Review by PatrykBuniX — APPROVED** [review]
looks good to me 👍

**Pull request merged**